Task1/Program.cs

```diff
@@ -10,7 +10,262 @@ class Program
+            //Calculator();
+            //RangeOfNumber();
+            //Translator();
+            //Premium();
+            AditionalCalculator();
+            Console.ReadKey();
+        }
+
+        public static void HourPassed(int seconds)
+        {
+            int hours = seconds / 3600;
+            Console.WriteLine($" in {seconds} seconds is {hours} hours");
+        }
+
+        public static void PlaceOfVariable(int a, int b, int c)
+        {
+            if (a < b && b < c)
+            {
+                Console.WriteLine("B is between A and C");
+            }
+            else
+                Console.WriteLine("B is not between A and C");
+        }
+
+        public static void IsOdd(int number)
+        {
+            int amountOfRank = Convert.ToString(number).Length;
+            if (number % 2 == 1 && amountOfRank == 3)
+            {
+                Console.WriteLine($"{number} is odd and has {amountOfRank} rank");
+            }
+            else
+                Console.WriteLine(" Statement is false");
+        }
+
+        public static void Calculation()
+        {
+            int x = 10;
+            int y = 12;
+            int z = 3;
+            x += y - x++ * z;
+            z = --x - y * 5;
+            y /= x + 5 % z;
+            z = x++ + y * 5;
+            x = y - x++ * z;
+            Console.WriteLine($"x ={x}, y={y}, z={z}");
+        }
+
+        public static void Calculator()
+        {
+            int operand1 = 5;
+            int operand2 = 2;
+            Console.WriteLine("Enter math operation:");
+            string sign = Console.ReadLine();
+            switch (sign)
+            {
+                case "+":
+                    Console.WriteLine($"{operand1} + {operand2} = {operand1 + operand2}");
+                    break;
+                case "-":
+                    Console.WriteLine($"{operand1} - {operand2} = {operand1 - operand2}");
+                    break;
+                case "*":
+                    Console.WriteLine($"{operand1} * {operand2} = {operand1 * operand2}");
+                    break;
+                case "/":
+                    if (operand2 == 0)
+                    {
+                        Console.WriteLine(" Division by zero is impossible");
+                    }
+                    else
+                        Console.WriteLine($"{operand1} / {operand2} = {((double)operand1 / operand2)}");
+                    break;
+                default:
+                    Console.WriteLine("Operation is undeclared");
+                    break;
+            }
+        }
+
+        public static void RangeOfNumber()
+        {
+            Console.WriteLine(" Enter number from 0 to 100");
+            int number = Convert.ToInt32(Console.ReadLine());
+            if (0 <= number && number <= 14)
+            {
+                Console.WriteLine("Number is in range from 0 to 14");
+            }
+            if (15 <= number && number <= 35 )
```

Comment:
number can't have a value that could belong to several ranges at the same time. So no need to check further conditions if some previous was passed.
So please use else if statements.